templates/prometheus.yaml

```diff
@@ -118,6 +118,38 @@ spec:
               - source_labels: [__meta_kubernetes_namespace]
                 target_label: namespace
                 action: replace
+            - job_name: 'kubernetes-keepalived'
```

Comment:
if keepalived is not used, will prometheus triggers alert about "some target is down"?

Reply:
no cuz there is no pod with tcp port 6161 in kube-system namespace